src/lib/components/event/event-summary-row.svelte

```diff
@@ -38,103 +53,209 @@
   import EventLink from './event-link.svelte';
   import MetadataDecoder from './metadata-decoder.svelte';
 
-  export let event: IterableEvent;
-  export let group: EventGroup | undefined = undefined;
-  export let initialItem: IterableEvent | undefined;
-  export let index = 0;
-  export let compact = false;
-  export let expanded = false;
-  export let typedError = false;
-  export let active = false;
-  export let onRowClick: () => void = () => {};
+  interface Props {
+    event: WorkflowEvent;
```

Comment:
I think this should stay as an IterableEvent. The event could be a pending activity or pending nexus operation, not just a WorkflowEvent. Let me know how I can help if you get stuck on type errors.